dali/operators/image/convolution/gaussian_blur.cc

```diff
@@ -62,15 +62,14 @@ The same input can be provided as per-sample tensors.
     .NumOutput(1)
     .AllowSequences()
     .SupportVolumetric()
-    .AddOptionalArg<int>(kWindowSizeArgName, "The diameter of the kernel.",
-                         std::vector<int>{0}, true)
+    .AddOptionalArg<int>(kWindowSizeArgName, "The diameter of the kernel.", std::vector<int>{0},
+                         true)
     .AddOptionalArg<float>(kSigmaArgName, "Sigma value for the Gaussian Kernel.",
                            std::vector<float>{0.f}, true)
-    .AddOptionalArg(
-        "dtype", R"code(Output data type.
+    .AddOptionalArg("dtype", R"code(Output data type.
 
 Supported type: `FLOAT`. If not set, the input type is used.)code",
-        DALI_NO_TYPE);
+                    DALI_NO_TYPE);
```

Comment:
Please revert - reformatting adds a lot of "changes" to this already large PR, making it even harder to review.